src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateImplicitNamespaceImports.targets

```diff
@@ -0,0 +1,57 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.GenerateImplicitNamespaceImports.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    ============================================================
+                                     GenerateImplicitNamespaceImports
+
+    Generates implicit namespace imports source to intermediate directory for C# projects
+    ============================================================
+    -->
+  <PropertyGroup>
+    <GeneratedImplicitNamespaceImportFile Condition="'$(GeneratedImplicitNamespaceImportFile)' ==''">$(IntermediateOutputPath)$(MSBuildProjectName).ImplicitNamespaceImports$(DefaultLanguageSourceExtension)</GeneratedImplicitNamespaceImportFile>
+    <DisableImplicitNamespaceImports_DotNet Condition="'$(DisableImplicitNamespaceImports_DotNet)' == '' and '$(DisableImplicitNamespaceImports)' != 'true'">false</DisableImplicitNamespaceImports_DotNet>
```

Comment:
I'm not aware of any common precedent for underscore separators in MSBuild property names. I get the pattern and I don't dislike it, but maybe we should loop in some more folks before establishing new idioms. /cc @rainersigwald @clairernovotny

Reply:
(that said, I'm not suggesting we should block this - we could discuss separately and change it later if necessary)

Reply:
Good point @mhutch. I had coined that on a whim in a chat somewhere and agree that it like deserves more discussion and thought before it sets any kind of precedent. Another case that comes to mind like this are the various default file globbing patterns the various SDKs add today.

@JunTaoLuo as @mhutch says let's not block on this now but it might be something we tweak before GA depending on result of discussions with MSBuild and other folk.

Reply:
Sounds good

Reply:
Let's fix the underscoring after this is merged